src/test/java/org/helpberkeley/memberdata/MainTest.java

```diff
@@ -570,6 +585,31 @@ private void completedOneKitchenOrders(String filepath) throws IOException {
         assertThat(statusPost.topic_id).isEqualTo(Constants.TOPIC_POST_COMPLETED_ONEKITCHEN_ORDERS.getId());
     }
 
+    private void completedOneKitchenOrdersWebURL(String filepath) throws IOException {
+        LocalDate yesterday = LocalDate.now(Constants.TIMEZONE).minusDays(1);
+        DateTimeFormatter format = DateTimeFormatter.ofPattern("yyyy/MM/dd");
+        String yesterdayStr = yesterday.format(format);
+
+        String completedOneKitchenOrdersRequest =
+                "{ \"success\": true, \"columns\": [ \"post_number\", \"deleted_at\", \"raw\" ], "
+                        + "\"rows\": [ "
+                        + "[ 42, null, \""
+                        + yesterdayStr
+                        + "\n[xyzzy.csv|attachment](https://go.helpberkeley.org/uploads/short-url/" + filepath + ") (5.8 KB)\" ] "
```

Comment:
Use Constants.WEB_CSV_PREFIX.  Avoid duplicating strings in code when possible to reduce brittleness (e.g. tests breaking when the constant is changed)